builtin/network.go

```diff
@@ -0,0 +1,63 @@
+package builtin
+
+import (
+	"fmt"
+
+	"github.com/filecoin-project/go-state-types/big"
+)
+
+// PARAM_SPEC
+// The duration of a chain epoch.
+// Motivation: It guarantees that a block is propagated and WinningPoSt can be successfully done in time all supported miners.
+// Usage: It is used for deriving epoch-denominated periods that are more naturally expressed in clock time.
+// TODO: In lieu of a real configuration mechanism for this value, we'd like to make it a var so that implementations
+// can override it at runtime. Doing so requires changing all the static references to it in this repo to go through
+// late-binding function calls, or they'll see the "wrong" value.
+// https://github.com/filecoin-project/specs-actors/issues/353
+// If EpochDurationSeconds is changed, update `BaselineExponent`, `lambda`, and // `expLamSubOne` in ./reward/reward_logic.go
+// You can re-calculate these constants by changing the epoch duration in ./reward/reward_calc.py and running it.
+const EpochDurationSeconds = 30
```

Comment:
how will this work for devnets etc?
Maybe we should have build tags?

Reply:
Whatever was already working here should keep working. Let's not bring in new scope to this PR which should be essentially a mechanical move/rename.

Reply:
Strong agree with @anorth